dkist/conftest.py

```diff
@@ -290,3 +292,14 @@ def small_visp_dataset():
     vispdir = Path(rootdir) / "small_visp"
     with asdf.open(vispdir / "test_visp.asdf") as f:
         return f.tree['dataset']
+
+
+@pytest.fixture(scope="session")
+def large_visp_dataset():
+    vispdir = Path(rootdir) / "large_visp"
+    ds = SimpleVISPDataset(n_maps=1, n_steps=20, n_stokes=4, time_delta=10,
+                           linewave=500*u.nm, detector_shape=(500, 1280))
```

Comment:
Make the detector shape smaller and it will take less time to save out the files, we don't really need the array shape to be realistic for anything do we?